build-caching-maven-samples/enforcer-project/pom.xml

```diff
@@ -128,118 +137,31 @@
                           <ignore>mojoExecution</ignore>
                         </ignoredProperties>
                       </inputs>
-                      <iteratedProperties>
-                        <property>
-                          <name>rules</name>
-                          <inputs>
-                            <properties>
-                              <property>
-                                <name>includes</name>
-                              </property>
-                              <property>
-                                <name>excludes</name>
-                              </property>
-                              <property>
-                                <name>searchTransitive</name>
-                              </property>
-                              <property>
-                                <name>message</name>
-                              </property>
-                            </properties>
-                          </inputs>
-                        </property>
-                      </iteratedProperties>
-                      <nestedProperties>
```

Comment:
It's a little unclear to me what this nested property definition was doing before, given that `project` is also defined for this execution as ignored.

Reply:
was this resolved?

Reply:
I believe so - I don't think `project` should have been ignored before. The `dependencies` property within `project` looks to influence the outcome of the `enforce-banned-dependencies` execution, so removed it from `ignoredProperties`.